duckdb_engine/__init__.py

```diff
@@ -670,6 +672,47 @@ def _comment_query(  # type: ignore[no-untyped-def]
                 )
 
 
+def _serial_sequence_name(table_name: str) -> str:
+    return f"{table_name}_pk_sequence"
```

Comment:
- If you have the same named table in two different schemas, do they both get the same name? This would be incorrect.
- This assumes that there is only a single SERIAL column per table. Is it possible to have multiple?

Reply:
- It should be possible, to include the schema in the name.
- So far, this only allows one SERIAL per table, yes. Postgres uses a naming scheme of "{table_name}_{column_name}_seq" for its SERIALs. If I remember correctly, I didn't find a good way to access the column name from within the parser.